utils/_context/containers.py

```diff
@@ -994,9 +998,10 @@ def __init__(self, host_log_folder) -> None:
 class MySqlContainer(SqlDbTestedContainer):
     def __init__(self, host_log_folder) -> None:
         super().__init__(
-            image_name="mysql/mysql-server:latest",
+            image_name="mysql/mysql-server:8.0.32",
             name="mysqldb",
-            command="--default-authentication-plugin=mysql_native_password",
+            command="--lc-messages-dir=/usr/share/mysql-8.0/english "
```

Comment:
out of curiosity, why this change ?

Reply:
I got an error at some point about missing messages, probably related to a different locale in my machine? I don't remember exactly.